js/data/move_data.js

```diff
@@ -94,7 +94,7 @@ var MOVES_RBY = {
 		type: 'Normal',
 		category: 'Physical',
 		makesContact: true,
-		hasRecoil: true
+		hasRecoil: 33
```

Comment:
I guess using Double-Edge as my example was a bad idea since its recoil is 25% before Gen 3.

Reply:
Whoops oh yeah lol